.github/workflows/baseSuiteE2E.yml

```diff
@@ -0,0 +1,246 @@
+name: Base E2E Test Suite for Minimum VSCode Version
+
+on:
+  workflow_run:
+    workflows:
+      - Nightly Build Develop
```

Comment:
So this test suite will be running every night. Will the person on trust keep an eye on it?

Reply:
Yes, that's the duty of the support person 🙂